src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/ApplicationInsightsEventListener.kt

```diff
@@ -0,0 +1,76 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.court
+
+import com.microsoft.applicationinsights.TelemetryClient
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkAppointmentSpecification
+import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkBookingSpecification
+import uk.gov.justice.digital.hmpps.whereabouts.dto.VideoLinkBookingUpdateSpecification
+import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkAppointment
+import uk.gov.justice.digital.hmpps.whereabouts.model.VideoLinkBooking
+import uk.gov.justice.digital.hmpps.whereabouts.security.AuthenticationFacade
+
+@Component
+class ApplicationInsightsEventListener(
```

Comment:
Maybe this class should have tests?

Reply:
Yes, you're right. The tests were in CourtService, but I forgot to move them over when I moved this code into another class.

Reply:
Added a test class